### ISSUE TITLE
Training and testing procedures are not the same

### ISSUE THREAD

**Issue by wuqiangch** [luoyetx/deep-landmark#9]
I have a question.In the training, the results of the previous level of training are not used for the latter level of training. However, in the test phase, the previous results are used  as a reference for the face position of next test phase. My understanding is like this. Is it right?

**Comment by luoyetx** [comment]
What if level-1 is failed, the data processed by level-1 are not stable, there might be some wrong data for level-2. Randomly sample around a point can cover the situation that a landmark is far away from the ground truth. However, if a landmark from level-1 is failed (too far away from the ground truth)m level-2 can not help.

**Comment by wuqiangch** [comment]
I have trained the nets several time and always got worse a liitle than your model's result  . Just because of  the network initialization?

**Comment by luoyetx** [comment]
maybe you should run more iterations or tuning the learning rate and other parameters during the training goes on.
